doc/memos/rdm0.md

```diff
@@ -0,0 +1,146 @@
+- RIOT Maintainers Community (RIOT-MC)  
+- Developer Memo: 0  
+- Author(s): Emmanuel Baccelli  
```

Comment:
I am not in favor of dropping and neither of putting contact info here for the reason that if there are many authors, adding all contact info for all authors here would be messy.
Much better to keep contact info at the end of the format as currently proposed, in my opinion.